modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -699,23 +699,38 @@ class ApiClient(object):
 
         return ss
 
-    def get_authorization_header(self, hdrs, signed_msg):
+    def get_authorization_header(self, signed_headers, signed_msg, created_ts):
         """
         Calculates and returns the value of the 'Authorization' header when signing HTTP requests.
         
-        :param hdrs : The list of signed HTTP Headers
-        :param signed_msg: Signed Digest
+        :param signed_headers : The list of signed HTTP Headers.
+        :param signed_msg: Signed Digest.
+        :param created_ts: The time when the signature was created, as a UNIX timestamp.
         :return: instance of digest object 
         """
 
+        i = 0
+        headers_value = ""
+        is_created_set = False
+        for key, _ in signed_headers.items():
+            key = key.lower()
+            if i > 0:
+                headers_value = headers_value + " "
+            headers_value = headers_value + key
+            if key == '(created)':
+                is_created_set = True
+            i += 1
+
         auth_str = ""
         auth_str = auth_str + "Signature"
 
         auth_str = auth_str + " " + "keyId=\"" + self.configuration.key_id + "\"," + "algorithm=\"" +
-            self.configuration.signing_scheme + "\"," + "headers=\"(request-target)"
+            self.configuration.signing_scheme + "\","
```

Comment:
Why do we have 3 lines defining auth_str?
Why not just use this?
`auth_str = "Signature keyId=\"{0}\",algorithm=\"{1}\"".format(self.configuration.key_id, self.configuration.signing_scheme)`

Reply:
simplifying, thanks.